File: appLab4B/src/main/java/com/github/hummel/cga/lab4b/Canvas.java
package com.github.hummel.cga.lab4b;

import javax.swing.*;
import java.util.List;

public class Canvas extends JFrame {

    public Canvas(List<Triangle> triangles) {
        setTitle("My frame");
        setDefaultCloseOperation(EXIT_ON_CLOSE);
        setSize(Main.width, Main.height);

        JPanel drawingPanel = new RenderPanel(triangles);
        drawingPanel.setSize(Main.width, Main.height);

        add(drawingPanel);
    }
}

File: appLab4B/src/main/kotlin/com/github/hummel/cga/lab4b/Canvas.kt
package com.github.hummel.cga.lab4b

import javax.swing.JFrame
import javax.swing.JPanel

class Canvas(faces: List<Face?>?) : JFrame() {
	init {
		title = "My frame"
		defaultCloseOperation = EXIT_ON_CLOSE
		setSize(Main.width, Main.height)

		val drawingPanel: JPanel = RenderPanel(faces)
		drawingPanel.setSize(Main.width, Main.height)

		add(drawingPanel)
	}
}
